python/paddle/v2/fluid/tests/book/test_recognize_digits.py

```diff
@@ -214,7 +226,8 @@ def inject_all_tests():
     for use_cuda in (False, True):
         for parallel in (False, True):
             for nn_type in ('mlp', 'conv'):
-                inject_test_method(use_cuda, parallel, nn_type)
+                for combine in (False, True):
```

Comment:
Please don't add this loop. This will twice the burden of CI. If there is not another way, we may only test the combined save/load.

Reply:
I added just 1 unit-test for the 'separate' params case.